src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch

```diff
@@ -859,7 +876,8 @@
       <sch:let name="languageTranslated_present" value="geonet:values-in($languageTranslated,
               ('eng', 'fra', 'spa', 'zxx'))"/>
 
-      <sch:let name="locMsgCt" value="geonet:prependLocaleMessage($loc/strings/ResourceDescriptionContentType, concat(gmd:CI_OnlineResource/gmd:linkage/gmd:URL, ' : '))" />
+      <sch:let name="resourceContentTypesList" value="geonet:resourceContentTypesList($thesaurusDir,$altLanguage2char)"/>
+      <sch:let name="locMsgCt" value="geonet:prependLocaleMessage(geonet:appendLocaleMessage($loc/strings/ResourceDescriptionContentType, $resourceContentTypesList),  concat(gmd:CI_OnlineResource/gmd:linkage/gmd:URL, ' : '))"/>
 
       <sch:assert test="($contentType = 'Web Service' or $contentType = 'Service Web' or
               $contentType = 'Dataset' or $contentType = 'Données' or
```

Comment:
If the message is generated using thesaurus values should this assert be checked using the thesaurus values as well rather than hardcoded? This way additional custom content types can be added.

Reply:
Updated to use thesaurus values in the assertion